14:09:11  Fixing /home/jenkins/b/workspace/ote/precommit/ote_sdk_tests_runner/src/ote_sdk/ote_sdk/configuration/elements/primitive_parameters.py
14:09:11  
14:09:11  isort (ote_cli|external).................................................Passed
14:09:13  black (ote_sdk|ote_cli)..................................................Failed
14:09:13  - hook id: black
14:09:13  - files were modified by this hook
14:09:13  
14:09:13  reformatted ote_sdk/ote_sdk/configuration/elements/utils.py
14:09:13  All done! ✨ 🍰 ✨
14:09:13  1 file reformatted, 185 files left unchanged.
14:09:13  
14:09:13  black (rest).............................................................Passed
14:09:15  flake8...................................................................Passed
14:09:16  prettier.................................................................Passed
14:09:20  mypy (ote_sdk)...........................................................Passed
14:09:20  mypy (ote_cli)...........................................................Passed
14:09:21  mypy (external)..........................................................Passed
14:10:00  pylint...................................................................Failed
14:10:00  - hook id: pylint
14:10:00  - exit code: 12
14:10:00  
14:10:00  ************* Module ote_sdk.configuration.elements.utils
14:10:00  ote_sdk/ote_sdk/configuration/elements/utils.py:186:4: W0613: Unused argument 'instance' (unused-argument)
14:10:00  ote_sdk/ote_sdk/configuration/elements/utils.py:213:12: R1701: Consider merging these isinstance calls to isinstance(value, (float, int)) (consider-merging-isinstance)
14:10:00  ote_sdk/ote_sdk/configuration/elements/utils.py:223:4: W0613: Unused argument 'instance' (unused-argument)

--- a/ote_sdk/ote_sdk/configuration/elements/primitive_parameters.py
+++ b/ote_sdk/ote_sdk/configuration/elements/primitive_parameters.py
@@ -41,13 +41,13 @@ from .metadata_keys import (
     WARNING,
 )
 from .utils import (
+    attr_strict_float_converter,
+    attr_strict_float_on_setattr,
+    attr_strict_int_validator,
     construct_attr_enum_selectable_converter,
     construct_attr_enum_selectable_onsetattr,
     construct_attr_selectable_validator,
     construct_attr_value_validator,
-    attr_strict_int_validator,
-    attr_strict_float_converter,
-    attr_strict_float_on_setattr
 )
 
 # pylint:disable=too-many-arguments

--- a/ote_sdk/ote_sdk/configuration/elements/utils.py
+++ b/ote_sdk/ote_sdk/configuration/elements/utils.py
@@ -183,9 +183,9 @@ def convert_string_to_id(id_string: Optional[Union[str, ID]]) -> ID:
 
 
 def attr_strict_int_validator(
-        instance: ParameterGroup,  # pylint: disable=unused-argument
-        attribute: Attribute,
-        value: int
+    instance: ParameterGroup,  # pylint: disable=unused-argument
+    attribute: Attribute,
+    value: int,
 ) -> None:
     """
     Validates that the value set for an attribute is an integer.
@@ -222,9 +222,9 @@ def _validate_and_convert_float(value: float) -> Optional[float]:
 
 
 def attr_strict_float_on_setattr(
-        instance: ParameterGroup,  # pylint: disable=unused-argument
-        attribute: Attribute,
-        value: float
+    instance: ParameterGroup,  # pylint: disable=unused-argument
+    attribute: Attribute,
+    value: float,
 ) -> float:
     """
     Validate that the value set for an attribute is a float, or a number that can be

--- a/ote_sdk/ote_sdk/configuration/elements/utils.py
+++ b/ote_sdk/ote_sdk/configuration/elements/utils.py
@@ -183,8 +183,10 @@ def convert_string_to_id(id_string: Optional[Union[str, ID]]) -> ID:
 
 
 def attr_strict_int_validator(
-        instance: ParameterGroup, attribute: Attribute, value: int
-) -> None:  # pylint: disable=unused-argument
+        instance: ParameterGroup,  # pylint: disable=unused-argument
+        attribute: Attribute,
+        value: int
+) -> None:
     """
     Validates that the value set for an attribute is an integer.
 
@@ -210,7 +212,7 @@ def _validate_and_convert_float(value: float) -> Optional[float]:
     :return: Value as float if value is valid, None otherwise
     """
     valid = True
-    if not (isinstance(value, float) or isinstance(value, int)):
+    if not isinstance(value, (float, int)):
         valid = False
     if isinstance(value, bool):
         valid = False
@@ -220,7 +222,9 @@ def _validate_and_convert_float(value: float) -> Optional[float]:
 
 
 def attr_strict_float_on_setattr(
-        instance: ParameterGroup, attribute: Attribute, value: float
+        instance: ParameterGroup,  # pylint: disable=unused-argument
+        attribute: Attribute,
+        value: float
 ) -> float:
     """
     Validate that the value set for an attribute is a float, or a number that can be

